tasks/test.js

```diff
@@ -6,6 +6,7 @@ module.exports = function (grunt) {
 
   grunt.registerTask('test:quick', [
     'test:server',
+    'test:ui',
     'test:browser'
```

Comment:
this isn't really your problem, but the naming here is weird. Both `test:ui` and `test:browser` run in the browser. We should probably change `test:browser` and `test:server` to something that describes that they're unit tests